ITEM.tsv

```diff
@@ -3097,9 +3097,9 @@ ITEM_20150317_003096	Supplies
 ITEM_20150317_003097	Supply Bag
 ITEM_20150317_003098	Portable supplies to be sent first to Danyus' squad. 
 ITEM_20150317_003099	Puragi's Hook
-ITEM_20150317_003100	A hook that soldier Roy has to collect. Roy can return to his troops as soon as he collects it.
+ITEM_20150317_003100	A hook that Roy the soldier has to collect. He can return to his troops as soon as he collects it.
```

Comment:
Leave soldier Roy as Soldier Roy. Roy the soldier would be implying the soldier is a title of his. His title is Soldier Roy not Roy the Soldier. This is just an assumption. But its cleaner.